packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/V1_ProtocolToMetaModelPropertyMappingBuilder.ts

```diff
@@ -190,6 +186,10 @@ export class V1_ProtocolToMetaModelPropertyMappingBuilder
         localMappingProperty.type,
       ).value;
     } else {
+      assertNonEmptyString(
```

Comment:
Because we are relying on Hardiks change, we need to coordinate when legend engine releases to merge this right?

Reply:
We are not relying on his change. His change is not gonna return `class` field for pure mapping and relational mapping. So i think these two changes are not dependent. We can merge this even before the engine release